tgui/packages/tgui/layouts/Window.js

```diff
@@ -26,37 +27,45 @@ export class Window extends Component {
     const {
       config,
       debugLayout,
+      standby,
     } = useBackend(this.context);
+    const { store } = this.context;
     // Determine when to show dimmer
     const showDimmer = config.observer
       ? config.status < UI_DISABLED
       : config.status < UI_INTERACTIVE;
+    const title = standby ? '' : decodeHtmlEntities(config.title);
     return (
       <Layout
         className="Window"
         theme={theme}>
         <TitleBar
           className="Window__titleBar"
-          title={decodeHtmlEntities(config.title)}
+          title={title}
           status={config.status}
           fancy={config.fancy}
           onDragStart={dragStartHandler}
           onClose={() => {
             logger.log('pressed close');
             releaseHeldKeys();
-            winset(config.window, 'is-visible', false);
-            runCommand(`uiclose ${config.ref}`);
+            callByond('', {
+              src: config.ref,
+              action: 'tgui:close',
+            });
+            store.dispatch(enterStandby());
           }} />
-        <div
-          className={classes([
-            'Window__rest',
-            debugLayout && 'debug-layout',
-          ])}>
-          {children}
-          {showDimmer && (
-            <div className="Window__dimmer" />
-          )}
-        </div>
+        {!standby ? (
```

Comment:
Preferrably should be `!standby && ()`, since it's shorter (and booleans do not render).